### ISSUE TITLE
Rectangle Breaks with Sequoia

### ISSUE THREAD

**Issue by 3DCoded** [rxhanson/Rectangle#1565]
- [ ] This is a new bug that hasn't been documented in a closed issue or in the Readme.

macOS version: Sequoia 15.3
Rectangle version: v.86 (92)
Logs if applicable (In Rectangle menu, hold option, "View Logging..."): 

```
Unable to obtain window id
Unable to obtain the accessibility element with the specified attribute at mouse location
```

When I updated to Sequoia 15.3, Rectangle stopped working. Window snapping or shortcuts doesn't work. 

Any help would be appreciated!

**Comment by 3DCoded** [comment]
It looks like this bug is intermittent. After I log out and log back in again, it *sometimes* works.

**Comment by rxhanson** [comment]
Sometimes macOS updates can cause a bit of flakiness with the Accessibility API that Rectangle relies on.

First, try rebooting one more time if you haven't done that after the update. Next, try resetting the Accessibility permissions for Rectangle.

1. Close Rectangle if it's running
2. Here in System Settings > Privacy & Security > Accessibility, first disable Rectangle, then remove it with the minus button. (in that order)
3. Launch Rectangle and enable settings for it as prompted.

If you're handy with the Terminal you can execute this terminal command instead of step 2:
```
tccutil reset All com.knollsoft.Rectangle
```

Sometimes it can be an issue with specific apps as well, but it sounds like this happens regardless of the app you're trying to snap?

**Comment by tobsecret** [comment]
>It looks like this bug is intermittent. After I log out and log back in again, it sometimes works.

Had a similar experience on Sonoma 14.7.1 just now after the update. Logging out and back in made it go back to normal.

**Comment by 3DCoded** [comment]
Thank you both for your replies. I'll try those next time Rectangle stops working again.

**Comment by 3DCoded** [comment]
The issue happened again and after logging out and in again it worked again (not a full ⌘↑Q, just a ⌘^Q)
